tests/test_plugin.py

```diff
@@ -33,3 +45,31 @@ def test_destroy(fxa_client, fxa_account):
     result = testdir.runpytest()
     result.assert_outcomes(error=1, passed=1)
     assert 'ClientError: Unknown account' in result.stdout.str()
+
+
+def test_commandline_email_option(testdir, monkeypatch, random_email):
+    monkeypatch.setenv('TEST_EMAIL', '{}'.format(random_email))
+    testdir.makepyfile("""
+        import os
+
+        import pytest
+
+        def test_account(fxa_account):
+            assert os.getenv('TEST_EMAIL') == fxa_account.email
+    """)
+    result = testdir.runpytest('--fxa-email', random_email)
+    result.assert_outcomes(passed=1)
+
+
+def test_fxa_email_env_variable(testdir, monkeypatch, random_email):
+    monkeypatch.setenv('FXA_EMAIL', '{}'.format(random_email))
+    testdir.makepyfile("""
+        import os
+
+        import pytest
+
+        def test_account(fxa_account):
+            assert os.getenv('FXA_EMAIL') == fxa_account.email
```

Comment:
As above, I'd use string interpolation here instead of taking the value from the environment variable.